doc/etc/config.ini

```diff
@@ -13,7 +13,7 @@
 ; port                  port to listen on
 ; data_dir              location of data directory
 ; minval                minimum hash responsibility range
-; maxval                maxmimum hash responsibility range
```

Comment:
I believe the `maxval` and `minval` settings are no longer needed anyway. @nemanja-boric-sociomantic these should be entirely removed, right?

Reply:
You're correct.